src/views/DatasetManager/components/DatasetList/DatasetList.js

```diff
@@ -95,48 +132,51 @@ export const DatasetList = () => {
   );
 
   const datasetListHeader = (
-    <Box sx={{ display: 'flex', justifyContent: 'space-between', px: 2 }}>
+    <Box sx={{ display: 'flex', justifyContent: 'space-between', p: 2 }}>
       <Typography variant="h6">Datasets</Typography>
       <CreateDatasetButton />
     </Box>
   );
 
   return (
-    <>
-      <List subheader={datasetListHeader}>
-        <Divider />
-        {sortedDatasetList.map((dataset) => (
-          <ListItemButton
-            key={dataset.id}
-            selected={dataset.id === currentDataset?.id}
-            onClick={(e) => selectDataset(dataset)}
-          >
-            <ListItem
-              secondaryAction={
-                <Box>
-                  <IconButton onClick={() => refreshDataset(dataset.id)}>
-                    <RefreshIcon />
-                  </IconButton>
-                  <IconButton onClick={(event) => askConfirmationToDeleteDialog(event, dataset.id)}>
-                    <DeleteForeverIcon />
-                  </IconButton>
-                </Box>
-              }
-              disableGutters
-              sx={{ pl: dataset.depth * 2 }}
+    <div>
+      <SearchBar label="Find..." onSearchChange={filterDatasets} icon={<SearchIcon />} className={classes.searchBar} />
```

Comment:
Do we need to translate the label `Find...`?

Reply:
Indeed, we need :facepalm: thanks ! :pray: